### PR TITLE
Editable docs

### PR DESCRIPTION
**Pull request by timdunn22**
## What does this PR do?
1. Makes it so that users can not annotate documents that are editable.

## How to test?
1. Login and go to http://cove-staging.herokuapp.com/documents and click on a document that is editable
2. You should not be able to annotate a document and should see no errors in the javascript console.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -45,7 +45,7 @@
   resources :users, only: [:index, :show, :edit]
 
   authenticated :user do
-    root :to => "anthologies#index"
+    root :to => "users#show"
```
Comment (jamiefolsom) [inline config/routes.rb:48]: This branch needs to be rebased, as master contains a new root route, of: `root :to => "anthologies#index"`, among other things.

**Review by jamiefolsom — CHANGES_REQUESTED** [review]
This branch needs to be rebased. See note.

File: config/routes.rb (hunk 1 above)
Comment (timdunn22) [reply config/routes.rb]: Yes this has been resolved now.

**Review by jamiefolsom — APPROVED** [review]

**Comment by jamiefolsom** [comment]
@timdunn22 looks good -- please merge to master.

**Pull request merged**